warehouse/query-core/src/test/java/datawave/query/jexl/nodes/ExceededOrThresholdMarkerJexlNodeTest.java

```diff
@@ -424,15 +431,49 @@ public void valueListTest() throws Exception {
         pointList.addAll(Arrays.asList(POINT_1, POINT_2, POINT_3, POINT_5, POINT_6, POINT_7, POINT_9, POINT_10, POINT_11));
         
         for (DefaultEvent event : events) {
-            String wkt = null;
+            List<String> wkt = new ArrayList<>();
             
             for (DefaultField field : event.getFields()) {
                 if (field.getName().equals(GEO_FIELD))
-                    wkt = field.getValueString();
+                    wkt.add(field.getValueString());
             }
             
             // ensure that this is one of the ingested events
-            Assert.assertTrue(pointList.remove(wkt));
+            Assert.assertTrue(pointList.removeAll(wkt));
+        }
+        
+        Assert.assertEquals(0, pointList.size());
+    }
+    
+    @Test
+    public void docSpecificValueListTest() throws Exception {
```

Comment:
Won't hold up an approval, but it might be nice to see a non-geo case that exercises this code path.

Reply:
Aside from the fact that these values happen to be geohashes, this isn't really a geo-specific test.  We aren't using any geo functions, so it's just a simple term equality evaluation being performed here.